ruby-lsp.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir.glob("lib/**/*.rb") + ["README.md", "VERSION", "LICENSE.txt"]
   s.bindir = "exe"
-  s.executables = ["ruby-lsp", "ruby-lsp-check"]
+  s.executables = ["ruby-lsp", "ruby-lsp-check", "ruby-lsp-index-info"]
```

Comment:
What might be a better name for this?

Reply:
How about `ruby-lsp-debug`?

Reply:
How about `ruby-lsp-doctor` (like `brew doctor`) to avoid any confusion with the `debug` gem?

Reply:
My thought was that it might become a more general purpose tool for troubleshooting setups in future.

Reply:
Yeah, I agree with that. We may want to expand that for other functionality and renaming the executable later may prove painful.

Reply:
I understand the argument but now it doesn't start the LSP server. If I saw it and assumed it's like `ruby-lsp` but with debugging output, I'd be very confused to see it just exits after indexing.